src/test/scala/uk/gov/nationalarchives/db/users/LambdaSpec.scala

```diff
@@ -60,20 +73,27 @@ class LambdaSpec extends AnyFlatSpec with Matchers {
     }
   }
 
-  def checkPrivileges(username: String, expectedPrivileges: List[String]) = {
+  def checkPrivileges(username: String, expectedPrivileges: List[String]): Assertion = {
     val privileges: List[String] = sql"SELECT privilege_type FROM information_schema.table_privileges WHERE grantee=$username"
       .map(rs => rs.string("privilege_type"))
       .list.apply()
     privileges.sorted should equal(expectedPrivileges)
   }
 
-  "The process method" should s"create the users with the correct parameters" in {
+  "The process method" should s"create the users with the correct parameters in the consignment database" in {
```

Comment:
Don't think you need string interpolation for the test desciption.

Reply:
Nope, not sure why that's there. I've taken it out.